### PR TITLE
STY: mighti clean updates

### PR DESCRIPTION
**Pull request by jklenzing**
Reduces code duplication and code complexity in the MIGHTI clean routines

### CODE REVIEW

**Comment by jklenzing** [comment]
Coverage changes consistent with reduction in code lines

File: pysatNASA/instruments/icon_mighti.py [hunk 1]
```diff
@@ -132,68 +132,77 @@ def clean(self):
...
-        if self.clean_level in ['clean', 'dusty']:
+            _clean_vars(wind_vars, wind_flag, min_val[self.clean_level])
+            _clean_vars(ver_vars, ver_flag, min_val[self.clean_level])
+
+        elif self.tag.find('temp') >= 0:
+            # neutral temperatures
+            var = 'Temperature'
+            saa_flag = 'Quality_Flag_South_Atlantic_Anomaly'
+            cal_flag = 'Quality_Flag_Bad_Calibration'
+            if saa_flag not in self.variables:
```
Comment (JonathonMSmith) [inline pysatNASA/instruments/icon_mighti.py:201]: Please excuse my icon ignorance, but can pysat load different data levels (L1, L2 etc) for icon? When is it expected to find that the flags need 'ICON_L1_MIGHTI' prepended to them?

**Review by JonathonMSmith — APPROVED** [review]
I think this looks neater.

File: pysatNASA/instruments/icon_mighti.py (hunk 1 above)
Comment (jklenzing) [reply pysatNASA/instruments/icon_mighti.py:201]: We don't have Level 1 data yet, but some of the data products copy over variables from the Level 1 files, which is why some variables have the prefix in the variable name

**Pull request merged**